src/mixins/filing-template-mixin.ts

```diff
@@ -315,11 +315,8 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
       this.parseStaffPayment(draftFiling)
     }
 
-    // if this is a premium account and Folio Number exists then restore it
-    if (this.isPremiumAccount) {
-      if (draftFiling.header.folioNumber) {
-        this.setFolioNumber(draftFiling.header.folioNumber)
-      }
+    if (draftFiling.header.folioNumber) {
+      this.setFolioNumber(draftFiling.header.folioNumber)
```

Comment:
Check with Yui whether folio number applies only to premium accounts.

Reply:
Okay. Will do

Reply:
Yui confirmed only Premium accounts. Applying changes now.

Reply:
Looks good. Resolving.